Formula/maturin.rb

```diff
@@ -0,0 +1,18 @@
+class Maturin < Formula
+  desc "Build and publish Rust crates as Python packages"
+  homepage "https://github.com/PyO3/maturin"
+  url "https://github.com/PyO3/maturin/archive/refs/tags/v0.10.6.tar.gz"
+  sha256 "e6a9a67cc62ffe248654e60e7ec211bf23319c4c936ad87022f7a1fd0997430d"
+  license "MIT"
+  head "https://github.com/PyO3/maturin.git"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match "Build and publish crates with pyo3", shell_output("#{bin}/maturin --help 2>&1")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Updated.